kDriveCore/Data/Cache/AccountManager.swift

```diff
@@ -434,4 +440,21 @@ public class AccountManager: RefreshTokenDelegate, AccountManageable {
     public func account(for userId: Int) -> Account? {
         return accounts.first { $0.userId == userId }
     }
+
+    public func switchToNextAvailableAccountOrLogout() {
+        Task { @MainActor in
```

Comment:
I need to run this on the main queue for navigation code to work well.
This is refactored on the APIV3 branch, so I won't refactor further here.